config/migration-snapshot.php

```diff
@@ -44,4 +44,16 @@
     |
     */
     'trim-underscores' => env('MIGRATION_SNAPSHOT_TRIM_UNDERSCORES', true),
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Include Data
+    |--------------------------------------------------------------------------
+    |
+    | Include existing table data in the database dump. Useful for when you
+    | have constant defined values like a system user with a specific ID or
+    | records with special IDs which must match another environment.
+    |
+    */
+    'include-data' => env('MIGRATION_SNAPSHOT_INCLUDE_DATA', false),
```

Comment:
This could probably be called `'data' =>` and `MIGRATION_SNAPSHOT_DATA`.

Reply:
Updated to use `data` and to also support PG / Sqlite